pkg/render/node_test.go

```diff
@@ -2681,6 +2681,90 @@ var _ = Describe("Node rendering tests", func() {
 				Expect(rtest.GetContainer(ds.Spec.Template.Spec.InitContainers, "install-cni").VolumeMounts).To(ConsistOf(expectedCNIVolumeMounts))
 			})
 
+			It("should render cni config with sysctl parameters", func() {
+				sysctl := []operatorv1.Sysctl{
+					{
+						Key:   "net.ipv4.tcp_keepalive_intvl",
+						Value: "15",
+					}, {
+						Key:   "net.ipv4.tcp_keepalive_probes",
+						Value: "6",
+					},
+					{
+						Key:   "net.ipv4.tcp_keepalive_time",
```

Comment:
Is there a reason this key should be missing in the `cniCM.Data["config"]` check below?

Reply:
do not need to check all, but to be in sync i added this as well